File: PHP/1008.php
<?php
$num = rtrim(fgets(STDIN));
$hours = rtrim(fgets(STDIN));
$percent = rtrim(fgets(STDIN));
echo "NUMBER = ".($num) . "\n";
echo "SALARY = U$ ".(number_format($hours*$percent,2,".","")) . "\n";
?>

File: Elixir/1008.exs
num = IO.gets("") |> String.trim() |> String.to_integer()
hours = IO.gets("") |> String.trim() |> String.to_integer()
percent = IO.gets("") |> String.trim() |> String.to_float()

IO.puts("NUMBER = #{num}")
IO.puts("SALARY = U$ #{(hours * percent) |> :erlang.float_to_binary(decimals: 2)}")
